ext/soap/soap.stub.php

```diff
@@ -604,7 +604,7 @@ public function __getLastRequestHeaders(): ?string {}
         public function __getLastResponseHeaders(): ?string {}
 
         /** @tentative-return-type */
-        public function __doRequest(string $request, string $location, string $action, int $version, bool $oneWay = false): ?string {}
+        public function __doRequest(string $request, string $location, string $action, int $version, bool $oneWay = false, ?string $uriParserClass = null): ?string {}
```

Comment:
Given the number of tests that you changed, is this a breaking change? If so, it needs to be noted in the RFC.

Reply:
yes, that's a good idea